css/style.css

```diff
@@ -0,0 +1,508 @@
+.container {
+  background: url(../img/Background.png) no-repeat;
+  background-size: cover;
+  display: flex;
+  max-width: 1440px;
+  min-width: 320px;
+  min-height: 1062px;
+  margin: auto;
+}
+
+.menu { 
+  margin: 130px 100px 0;
+}
+
+.title {
+  color: #FFF;
+  margin-bottom: 10px;
+  font: normal bold 57px/67px "Roboto";
+}
+
+.subtitle {
+  color: #FFF;
+  margin: 0 0 60px 0;
+  font: normal bold 19px/22px "Roboto";
+}
+
+.menu__select-level {
+  margin-bottom: 113px;
+  position: relative;
+}
+
+.level_btn {
+  display: flex;
+  margin-bottom: 23px;
+  position: relative;
+  background: none;
+  border: none;
+  color: #FFF;
+  font: normal bold 50px/59px "Roboto";
+  outline: none;
+  padding-left: 48px;
+}
+
+.level_btn:hover {
+  opacity: 0.8;
+  cursor: pointer;
+  position: relative;
+}
+
+.level_active:before {
+  content: '';
+  background-color: #D62B2E;
```

Comment:
Codacy found an issue: [Expected "#D62B2E" to be "#d62b2e" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Olga_FinalGame_JS/pullRequest?prid=5821092)